test/mocha.opts

```diff
@@ -1,4 +1,5 @@
 --reporter dot
 --ui bdd
 --slow 50
---timeout 200
\ No newline at end of file
+--timeout 200
+--exit
```

Comment:
Reminder that we talked about leaving a comment to specify why this is necessary